sklearn/feature_selection/univariate_selection.py

```diff
@@ -232,12 +232,12 @@ def f_regression(X, y, center=True):
     Quick linear model for testing the effect of a single regressor,
     sequentially for many regressors.
 
-    This is done in 3 steps:
+    This is done in 2 steps:
 
-    1. The regressor of interest and the data are orthogonalized
-       wrt constant regressors.
-    2. The cross correlation between data and regressors is computed.
-    3. It is converted to an F score then to a p-value.
+    1. The cross correlation between data and regressors is computed
```

Comment:
Yeah even I would prefer the first one because of the same reasons. I'll make the changes.